src/main/java/com/derongan/minecraft/deeperworld/extensions/DeeperWorldExtensions.kt

```diff
@@ -27,7 +27,11 @@ internal fun Entity.getPassengersRecursive(): List<Entity> {
 }
 
 internal fun Player.getLeashedEntities(): List<LivingEntity> {
-    return getNearbyEntities(20.0, 20.0, 20.0) // Max leashed entity range is 10 blocks, therefore these parameter values
```

Comment:
I think put the comment on the line above, then the auto formatter won't try so aggressively to make it all fit on one line, it looked cleaner before.